src/utils.ts

```diff
@@ -0,0 +1,18 @@
+import * as os from 'os';
+import * as path from 'path';
+import * as rimraf from 'rimraf';
+import * as uniqid from 'uniqid';
+
+export function del(filename: string) {
+    return new Promise((resolve, reject) => rimraf(filename, (err) => {
+        if (err) {
+            reject(err);
+        }
+
+        resolve();
```

Comment:
You're _always_ resolving resolving the promise. I'll remove it when writing the test for it

Reply:
Ah yeah, kind of forgot that it isn't the same as returning :S

Reply:
😮 Even with returning I would put it in the else for clarity 🙄

Reply:
Didn't you say less code is better? :P

Reply:
When you're talking about control statements, you're getting into clean code territory.